ui/src/app/pages/admin/admin-notebook-view.tsx

```diff
@@ -0,0 +1,110 @@
+import {SpinnerOverlay} from 'app/components/spinners';
+import {workspaceAdminApi} from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import {reactStyles} from 'app/utils';
+import {reactRouterUrlSearchParams} from 'app/utils/navigation';
+import * as React from 'react';
+import {useEffect, useState} from 'react';
+import {useParams} from 'react-router';
+
+const styles = reactStyles({
+  heading: {
+    color: colors.primary,
+    fontSize: 16,
+    fontWeight: 500,
+  },
+  notebook: {
+    width: '100%',
+    height: 'calc(100% - 40px)',
+    position: 'absolute',
+    border: 0
+  },
+  error: {
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginTop: '56px',
+    background: colors.warning,
+    border: '1px solid #ebafa6',
+    borderRadius: '5px',
+    color: colors.white,
+    display: 'flex',
+    fontSize: '14px',
+    fontWeight: 500,
+    maxWidth: '550px',
+    padding: '8px',
+    textAlign: 'left'
+  },
+});
+
+interface Props {
+  workspaceNamespace: string;
+  nbName: string;
+  accessReason: string;
+}
+
+const AdminNotebookViewComponent = (props: Props) => {
+  const {workspaceNamespace, nbName, accessReason} = props;
+  const [notebookHtml, setHtml] = useState(undefined);
+  const [workspaceName, setWorkspaceName] = useState(undefined);
+  const [errorMessage, setErrorMessage] = useState(undefined);
+
+  const Header = () => {
+    const location = workspaceName ? `Workspace ${workspaceNamespace}/${workspaceName}` : workspaceNamespace;
+    const link = <a href={`/admin/workspaces/${workspaceNamespace}`}>{location}</a>;
+
+    return <div style={styles.heading}>Viewing {nbName} in {link} for reason: {accessReason}</div>;
```

Comment:
nit: Probably want to split the reason onto another line. Can come later with a styling follow-on.